tap_github/__init__.py

```diff
@@ -2877,7 +2891,12 @@ def schemaSortFunc(val):
 
                 # sync stream
                 if not sub_stream_ids:
-                    state = sync_func(stream_schema, repo, state, mdata, start_date)
+                    if stream_id == 'commit_files' or stream_id == 'commit_files_meta':
```

Comment:
`commit_files` and `commit_files_meta` have substreams, right?  This won't ever be hit currently.   If we remove refs as sub streams of either stream, this will come into play, right?

I'm not positive the `stream_schemas` construction is necessary, but it should be fine.